Elmah.Io.Extensions.Logging/ElmahIoLoggerProvider.cs

```diff
@@ -7,16 +7,41 @@ public class ElmahIoLoggerProvider : ILoggerProvider
     {
         private readonly string _apiKey;
         private readonly Guid _logId;
+        private readonly FilterLoggerSettings _filter;
 
-        public ElmahIoLoggerProvider(string apiKey, Guid logId)
+        public ElmahIoLoggerProvider(string apiKey, Guid logId, FilterLoggerSettings filter = null)
         {
             _apiKey = apiKey;
             _logId = logId;
+
+            if (filter == null)
+            {
+                filter = new FilterLoggerSettings
+                {
+                    {"*", LogLevel.Trace}
```

Comment:
Maybe we should change it now and bump the version number a bit? Would that be ok for you? Not a lot of people are using this yet, why I think making this change now will be better than later.